otter/test/test_util.py

```diff
@@ -289,30 +292,34 @@ def test_apierror_identity(self):
         body = json.dumps({"identityFault": {"message": "ba"}})
         apie = APIError(410, body, {})
         err = UpstreamError(Failure(apie), 'identity', 'stuff', 'xkcd.com')
-        self.assertEqual(str(err), 'identity error: 410 - ba')
+        self.assertEqual(str(err), 'identity error: 410 - ba (stuff)')
         self.assertEqual(err.details, {
             'system': 'identity', 'operation': 'stuff', 'url': 'xkcd.com',
             'message': 'ba', 'code': 410, 'body': body, 'headers': {}})
 
     def test_apierror_unparsed(self):
         """
-        Wraps APIError from identity and uses default string if unable to parses
-        error body
+        Wraps APIError from identity and uses default string if unable to
+        parses error body
```

Comment:
Sorry nitpick from my earlier code: This should be `if unable to parse error body` :)